BUILD.md

```diff
@@ -341,56 +333,78 @@ index 7dfa97c..6d99e71 100644
 
  set_property(TARGET LLVMSupport PROPERTY LLVM_SYSTEM_LIBS "${system_libs}")
 ```
+   * Linux
+   Download llvm source code 6.0.1 and unzip to /llvm/source/path, then install to /llvm/install/path
+   ```
+   cd /llvm/source/path
+   mkdir build
+   cd build
+   cmake .. -DLLVM_TARGETS_TO_BUILD=X86 -DCMAKE_BUILD_TYPE=Release
+   cmake --build.
+   cmake -DCMAKE_INSTALL_PREFIX=/llvm/install/path -DBUILD_TYPE=Release -P cmake_install.cmake
+   ```
 
-Linux:
-```
-# download llvm source code 9.0.0 and unzip to /llvm/source/path, then install to /llvm/install/path
-cd /llvm/source/path
-mkdir build
-cd build
-cmake .. -DLLVM_TARGETS_TO_BUILD=X86 -DCMAKE_BUILD_TYPE=Release
-cmake --build.
-cmake -DCMAKE_INSTALL_PREFIX=/llvm/install/path -DBUILD_TYPE=Release -P cmake_install.cmake
-```
-
-Then you can build from source by using following command from the onnxruntime directory:
-Windows:
+#### Build Instructions
+##### Windows
 ```
 build.bat --use_tvm --use_llvm --llvm_path=\llvm\install\path\lib\cmake\llvm --use_mklml --use_nuphar --build_shared_lib --build_csharp --enable_pybind --config=Release
 ```
 
-Linux:
+##### Linux:
 ```
 ./build.sh --use_tvm --use_llvm --llvm_path=/llvm/install/path/lib/cmake/llvm --use_mklml --use_nuphar --build_shared_lib --build_csharp --enable_pybind --config=Release
 ```
 
+Dockerfile instructions are available [here](https://github.com/microsoft/onnxruntime/tree/master/dockerfiles#nuphar-public-preview)
+
+#### Notes
+* These instructions build the release flavor. The Debug build of LLVM would be needed to build with the Debug flavor of ONNX Runtime.
```

Comment:
>The Debug build of LLVM would be needed to build with the Debug flavor of ONNX Runtime [](start = 47, length = 86)

This is only for Windows build. Linux ORT debug build can link with release build of LLVM.